build.gradle

```diff
@@ -1,14 +1,8 @@
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
-    }
+plugins {
+    id 'org.openjfx.javafxplugin' version '0.0.7'

```

Comment:
The root project probably doesn't require the plugin? We might want to add `apply false` in that case.